src/journal/ObjectPlayer.cc

```diff
@@ -193,10 +193,10 @@ int ObjectPlayer::handle_fetch_complete(int r, const bufferlist &bl,
             ldout(m_cct, 20) << ": partial pad detected, will re-fetch"
                              << dendl;
           }
-          break;
```

Comment:
Should you add a break between lines 194-195? It seems like that wouldn't be a fatal error (potentially).

Reply:
Thanks. I fixed it with the `else` clause. Also, I removed `if (!invalid) ` check above as it does not look necessary now.